configuration.rst

```diff
@@ -497,56 +506,62 @@ For more information about local configuration, see :ref:`Local configuration of
-* | If ``<project_root>/env/java_ci`` is present, Gauge sets the environment variables mentioned in the corresponding ``java_ci`` environment. 
+* If ``java_ci`` is **not** found, Gauge ends with a non-zero exit code.
+* | If ``java_ci`` is present, Gauge sets the environment variables mentioned in the ``java_ci`` environment. 
   | Gauge then loads other environment variables from the ``default`` environment which are not yet set. 
-  | Finally, Gauge sets the remaining (if any) environment variables with values mentioned as per the table.
+  | If the ``default`` environment is not present, Gauge internally sets the environment variables, which is the same as those in ``default.properties``.
 
 Workflow : User runs ``gauge_reports_dir=newReportsDir gauge run specs``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 .. note::
-   User can also explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run specs``
-    
-Gauge sets all the environment variables from ``<project_root>/env/default`` and then the values from the table, except for ``gauge_reports_dir``. 
+   Alternately, user can explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run specs``.
+
+``default`` environment is located at <project_root>/env/.
+
+| Gauge sets all the environment variables from ``default``. 
+| If ``default`` is not present, Gauge internally sets the environment variables, which is the same as those in ``default.properities``, except for ``gauge_reports_dir``. 
+
 The value of ``gauge_reports_dir`` continues to be ``newReportsDir``.
 
 Workflow : User runs ``gauge_reports_dir=newReportsDir gauge run --env=java_ci specs``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``java_ci`` is the user-created environment at ``<project_root>/env``.
+
+.. note::
+   Alternately, user can explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run --env java_ci specs``
+
+Gauge sets the environment variables as mentioned in the ``java_ci`` environment except for ``gauge_reports_dir``.
+The value of ``gauge_reports_dir`` continues to be ``newReportsDir``.
+
+Workflow : User runs ``gauge_reports_dir=newReportsDir gauge run --env “java_ci, experimental” specs``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``java_ci, experimental`` are the user-created environments at ``<project_root>/env``.
 
 .. note::
-   User can also explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run --env java_ci specs``
+   Alternately, user can explicitly set ``gauge_reports_dir=newReportsDir`` in the operating system's user shell and then run ``gauge run --env "java_ci, experimental" specs``
```

Comment:
There is a tab or space at the start of this para, is that intentional?

Reply:
Hmm.. I am not able to see anything unusual in the file. I even ran the build and viewed the web page for this file. It appears fine. I have sent screen capture. The content above is within the note.